Mail/Views/Sync Profile/SyncCopyPasswordView.swift

```diff
@@ -0,0 +1,114 @@
+/*
+ Infomaniak Mail - iOS App
+ Copyright (C) 2022 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import InfomaniakDI
+import MailCore
+import MailResources
+import SwiftUI
+
+struct SyncCopyPasswordView: View {
+    @LazyInjectService private var snackbarPresenter: SnackBarPresentable
+
+    @EnvironmentObject private var mailboxManager: MailboxManager
+
+    @State private var applicationPassword: String?
+
+    @Binding var navigationPath: [SyncProfileStep]
+
+    var body: some View {
+        VStack(spacing: UIPadding.regular) {
+            Text(MailResourcesStrings.Localizable.syncTutorialCopyPasswordTitle)
+                .textStyle(.header2)
+                .multilineTextAlignment(.center)
+
+            VStack(spacing: UIPadding.large) {
+                MailResourcesAsset.lockIllustration.swiftUIImage
+
+                if let applicationPassword {
+                    HStack {
```

Comment:
Can we add a spacing? So if the password is long, it will still be spaced from the button